microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -38,227 +56,269 @@ To use experimental APIs, download a prerelease version of the WebView2 SDK from
+   ```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID. Replace the sample path with the path to the desired Microsoft Edge preview channel.
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+##### [Environment variable](#tab/environment-variable)
 
-To delete the `ReleaseChannelPreference` registry override, run the command:
+Name: `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`  
+Value: `<path to desired Microsoft Edge preview channel>`
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /f`
+The environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.
 
 
-<!-- 3. Environment Variable ============================================== -->
-## Using an environment variable
+##### [Group policy](#tab/group-policy)
 
-To make your application use a Microsoft Edge preview channel by using an environment variable:
+Set the `BrowserExecutableFolder` policy.
 
-1. In the Windows search bar, enter "environment", and then select **Edit the system environment variables**.
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`
+Value: `<path to desired Microsoft Edge preview channel>`
 
-   ![Using the Windows search bar to find where to edit environment variables](./set-preview-channel-images/search-bar-edit-sys-env-vars.png)
+Do either of the following:
 
-1. In the **System Properties** dialog, select the **Advanced** tab, and then select the **Environment Variables** button.
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  See [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge).
 
-   ![The Environment Variables button in the System Properties dialog](./set-preview-channel-images/system-properties-env-vars.png)
+* Use the built-in policy on Intune.  See [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge).
+
+---
+
+
+<!-- ====================================================================== -->
+## FAQs
+
+
+<!-- ------------------------------ -->
+#### When does the new runtime get picked up?
+
+If you are using an environment variable, registry key, or group policy, they are only applied to any new processes that are created; they do not apply to processes that are already running.  To ensure that all processes use the environment variable, registry key, or group policy, you would need to restart the processes.
```

Comment:
```suggestion
If you are using an environment variable, registry key, or group policy, they are only applied to any new WebView2 Runtime browser processes that are created; they do not apply to WebView2 Runtime browser processes that are already running. For example, if you already have a WebView2 with a specific user data folder (and therefore a corresponding browser process) and change ChannelSearchKind before attempting to create a new WebView2 using that same user data folder, it will use the existing WebView2 Runtime browser process, and not search for a different runtime.  To ensure that all processes use the environment variable, registry key, or group policy, you would need to restart the app, recreate the WebView2 controls, or create a new WebView2 that has a different user data folder than any other currently running WebView2 controls.
```